hypha/settings/base.py

```diff
@@ -155,6 +155,7 @@
 # Require an applicant to view their rendered application before submitting
 SUBMISSION_PREVIEW_REQUIRED = env.bool("SUBMISSION_PREVIEW_REQUIRED", True)
 
+
```

Comment:
Can we clear out this please.

Reply:
I did it in a later commit because I assume these get squashed. If I need to amend, let me know.